recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/hparams/pretrain_encoder.yaml

```diff
@@ -0,0 +1,194 @@
+# ############################################################################
+# Model: NAR-BERT-ASR-Encoder
+# https://arxiv.org/pdf/2104.04805.pdf
```

Comment:
Feel free to add Authors: your name; It's your work after all :-) Easier for us to know who to ping later as well.